swimster-api/models/Image.js

```diff
@@ -0,0 +1,134 @@
+const db = require('../db')
+const { uploadImageToS3, fetchImageUrlFromS3 } = require('../s3')
+const { UnauthorizedError, BadRequestError, NotFoundError } = require('../utils/error')
+
+class Image {
+    static publicImageInfo(image, url) {
+        return {
+            id: image.id,
+            image_name: image.image_name,
+            image_key: image.image_key,
+            image_size: image.image_size,
+            image_mimetype: image.image_mimetype,
+            listing_id: image.listing_id,
+            is_main_image: image.is_main_image,
+            image_url: url,
+        }
+    }
+
+    static async createImages(imageFiles, listingId) {
+        if (imageFiles.length < 5) {
```

Comment:
what happens if no files are uploaded?

Reply:
if no files are uploaded, it will still go through the exception. This will get handled in a snackbar

Reply:
Actually, it's still creating the listing because the error happens in Express. Will get error-handling in another PR very soon